Formula/zork.rb

```diff
@@ -0,0 +1,20 @@
+class Zork < Formula
+  desc "The Dungeon modified from FORTRAN to C"
+  homepage "https://github.com/devshane/zork"
+  url "https://github.com/devshane/zork/archive/v1.0.2.tar.gz"
+  sha256 "169e1848b1e3c503591c23ad4e66ce45e1d5ae617831634e1da9c8fca659e283"
+  head "https://github.com/devshane/zork.git"
+
+  def install
+    system "make", "DATADIR=#{share}", "BINDIR=#{bin}"
+    system "make", "install", "DATADIR=#{share}", "BINDIR=#{bin}", "MANDIR=#{man}"
+  end
+
+  test do
+    test_phrase = "Welcome to Dungeon.\t\t\tThis version created 11-MAR-91.\n"\
```

Comment:
Gave it a shot, the trailing newline of the Heredoc was throwing off the test so, I append a `<` after. Let me know what you thing!